openapi.yaml

```diff
@@ -4191,9 +4235,10 @@ components:
             $ref: '#/components/schemas/link'
     stac_version:
       type: string
-      description: The STAC version implemented.
-      enum:
-        - 0.9.0
+      description: >-
+        The [version of the STAC specification](https://github.com/radiantearth/stac-spec/releases),
+        which MAY not be equal to the [STAC API version](#section/STAC).
+        Supports versions 0.9.x and 1.x.x.
```

Comment:
just out of curiosity: is there a guarantee that 1.x.x will strictly follow 0.9.x, without a 0.10.x in between?

Reply:
Only my word as core contributor, but 1.0.0-beta is already released, too ;-)